content/docs/user-guide/troubleshooting.md

```diff
@@ -28,3 +28,23 @@ To solve this, it's often possible to increase the open file descriptors limit,
 with `ulimit` on UNIX-like system (for example `ulimit -n 1024`), or
 [increasing Handles limit](https://blogs.technet.microsoft.com/markrussinovich/2009/09/29/pushing-the-limits-of-windows-handles/)
 on Windows. Otherwise, please try using a lower `JOBS` value.
+
+## Unable to find credentials {#no-credentials}
+
+Make sure that you have your AWS credentials setup either through
+[usual AWS configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
+or using [DVC configs](/doc/command-reference/remote/modify).
+
+## Unable to connect {#connection-error}
+
+Make sure you have a working internet connection and you are able to access your
```

Comment:
```suggestion
Make sure you are online and able to access your
```

Reply:
(and this one since a similar suggestion was already applied in the core repo PR)